lib/oelite/meta/meta.py

```diff
@@ -436,15 +436,18 @@ def dump_var(self, key, o=sys.__stdout__, pretty=True, dynvars=[],
             expand = FULL_EXPANSION
         if not expand and func != "python":
             expand = OVERRIDES_EXPANSION
-        val = self.get(key, expand)
+        val,deps = self._get(key, expand)
 
         if not val:
             return 0
 
         val = str(val)
 
-        for dynvar_name, dynvar_val in dynvars:
-            val = string.replace(val, dynvar_val, dynvar_name)
+        if deps:
+            for dynvar_name, dynvar_token, dynvar_val in dynvars:
+                if dynvar_name not in deps:
+                    continue
+                val = string.replace(val, dynvar_val, dynvar_token)
```

Comment:
am I reading this right, that with this change e.g. SIGNATURE_FILE will no longer become a part of the hash? if so, that does not seam like a good idea :)